### ISSUE TITLE
resource links to directory incorrectly parsed

### ISSUE THREAD

**Issue by ransomts** [Kungsgeten/org-brain#297]
When I insert a link to a directory as a resource, it gets incorrectly opened from the visualize buffer. The org link acts correctly in the brain.org file. The issue seems to be based on a whitespace parsing error as the directory in question is similar to this: `~/Documents/Course Work/8250` and brain is opening a buffer called Course.

**Comment by Kungsgeten** [comment]
Hi! I recognize the problem but I though it was fixed (see #280). I tested it now and seems to work for me... Are you sure you're using the latest version of `org-brain`?

**Comment by ransomts** [comment]
That fixed it thanks! I was on v0.91, sorry for the double report!

**Comment by Kungsgeten** [comment]
No problem :) Glad that it wasn't a new problem 😅

**Comment by ransomts** [comment]
Hey this issue popped back up for me, so I looked around a bit and am writing this down in case somebody else needs it.
Here's the versions of relevant packages (these are what package-install pulls at the moment)

org-brain : 0.94
org-mode : 9.3.6
helm : 3.6.2

I don't think that I have anything in my org-brain configuration that would cause this but I can add that if you think of something that would help.

Basically the value being passed to `org-link-open-from-string` in `org-brain-open-resource` isn't formatted with the surrounding brackets for some reason. Could be a helm thing, that's the only completion I've tested? The fix for #280 adds them for an insert but this is for reading the resource. A similar insert to the open resource function passes all my tests, but I'm not sure if this would break other things (doesn't break my other links afaict). I can open a pr but it's such a small edit I thought I'd just drop it here.

https://github.com/Kungsgeten/org-brain/blob/6b2ec93ec92b4a244af9d600433e0d0a02eb5f1e/org-brain.el#L1294
`(org-link-open-from-string (format "[[%s]]" (org-brain--choose-resource...`
